src/main/resources/static/app.js

```diff
@@ -141,9 +145,20 @@ $(function () {
     }),
 
     $('#send').click(function () {
-        sendMessage()
+        if($('#message').val() !== ""){
```

Comment:
why are we validating the message field again here? Aren't we doing that in `Line 153`?